src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -43,38 +43,38 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
   font-family: "Source Sans Pro";
   font-style: normal;
   font-weight: 300;
-  src: url(/generated/sourcesanspro-light-webfont.eot?#iefix) format("embedded-opentype"),
-    url(/generated/sourcesanspro-light-webfont.woff2) format("woff2"),
-    url(/generated/sourcesanspro-light-webfont.woff) format("woff"),
-    url(/generated/sourcesanspro-light-webfont.ttf) format("truetype");
+  src: url(#{$font-path}/sourcesanspro-light-webfont.eot?#iefix) format("embedded-opentype"),
+    url(#{$font-path}/sourcesanspro-light-webfont.woff2) format("woff2"),
+    url(#{$font-path}/sourcesanspro-light-webfont.woff) format("woff"),
+    url(#{$font-path}/sourcesanspro-light-webfont.ttf) format("truetype");
 }
 
 @font-face {
   font-family: "Source Sans Pro";
   font-style: normal;
   font-weight: 400;
-  src: url(/generated/sourcesanspro-regular-webfont.eot?#iefix) format("embedded-opentype"),
-    url(/generated/sourcesanspro-regular-webfont.woff) format("woff"),
-    url(/generated/sourcesanspro-regular-webfont.ttf) format("truetype");
+  src: url(#{$font-path}/sourcesanspro-regular-webfont.eot?#iefix) format("embedded-opentype"),
```

Comment:
we had the `$font-path` defined but weren't using it. This fixes the localhost issue mentioned in the previous comment- the site pulls from a `/generated/hashed-font-name` like it does on the prod site.